config/crds/cdap_v1alpha1_cdapmaster.yaml

```diff
@@ -370,12 +370,25 @@ spec:
                 - status
                 type: object
               type: array
+            oldImage:
```

Comment:
What is this for?

Reply:
It's used by the templates to keep the image from changing during upgrade until the job is completed.